src/Glyphicon.js

```diff
@@ -1,13 +1,280 @@
 import React from 'react';
 import classNames from 'classnames';
-import BootstrapMixin from './BootstrapMixin';
-import styleMaps from './styleMaps';
 
 const Glyphicon = React.createClass({
-  mixins: [BootstrapMixin],
-
   propTypes: {
-    glyph: React.PropTypes.oneOf(styleMaps.GLYPHS).isRequired
+    /**
+     * bootstrap className
+     * @private
+     */
+    bsClass: React.PropTypes.string,
+    /**
+     * The part after dash of a glyphicon from the list
+     * http://getbootstrap.com/components/#glyphicons
+     * E.g. `euro` for the `glyphicon-euro`
+     * @type {("a glyphicon name's part after dash")}
```

Comment:
I _think_ if you remove the `(` and `)` around the type here it will just display the text which will read better.

I'd also change the wording to just: "an icon name' 

the examples should be clear that that means "after the dash"